talentmap_api/fsbid/services/post_access.py

```diff
@@ -173,3 +175,39 @@ def format_request_post_data_to_string(request_values, table_key):
     result_string = "{" + ",".join(data_entries) + "}"
     return result_string
 
+
+
+def grant_post_access_permissions(jwt_token, request):
+    '''
+    Grant Access for a Post
+    '''
+    args = {
+        "proc_name": 'prc_add_org_access',
+        "package_name": 'PKG_WEBAPI_WRAP_SPRINT99',
+        "request_body": request,
+        "request_mapping_function": map_grant_access_request,
+        "response_mapping_function": None,
+        "jwt_token": jwt_token,
+
+    }
+    return services.send_post_back_office(
+        **args
+    )
+
+def map_grant_access_request(req):
```

Comment:
```suggestion
def grant_post_access_permissions_req_mapping(req):
```

we are testing out this new naming convention in hopes it is easier to understand